Tests/SentryTests/PrivateSentrySDKOnlyTests.swift

```diff
@@ -114,6 +114,11 @@ class PrivateSentrySDKOnlyTests: XCTestCase {
         let itemData = "{}\n{\"length\":0,\"type\":\"attachment\"}\n".data(using: .utf8)!
         XCTAssertNotNil(PrivateSentrySDKOnly.envelope(with: itemData))
     }
+    
+    func testEnvelopeWithDataLengthGtZero() throws {
+        let itemData = "{}\n{\"length\":1,\"type\":\"attachment\"}\n".data(using: .utf8)!
+        XCTAssertNil(PrivateSentrySDKOnly.envelope(with: itemData))
```

Comment:
I expect this to be `nil`, unsure why the `length: 1` test above expects not nil

Reply:
Because the first states the attachment is empty, which is a valid envelope. Here, we state that the attachment has a length of 1, but there are no bytes for it. So it's an invalid envelope